index.html

```diff
@@ -0,0 +1,156 @@
+        <p class="secundary-paragraf">The Wrangler NFR is the richest and most prestigious rodeo in the world and an event like no other, an extravaganza for fans to enjoy the roping and riding activity of today’s cowboys. It is the culmination of the rodeo season where the top 15 contestants in bareback riding, steer wrestling, team roping, saddle bronc riding, tie-down roping, barrel racing and bull riding compete to take home top honors and their share of a multi-million dollar purse.</p>
+        <p class="date">2023.12.09(SAT) - 16(SAT)</p>
+      </div>
+    </section>
+
+  <!----------------------------------Events------------------------------------------->
+
+    <section class="events" id="events">
+      <div class="event-container">
+        <h2>Events</h2>
+        <div class="red-line"></div>
+
+        <ul>
+          <li>
+            <div class="event-li">
+              <img src="Capstone-img/table.png">
+              <p class="event">Saddle Bronc</p>
+              <p class="event-comment">A rodeo event where a rider uses a specialized saddle to stay on a bucking horse.</p>
+            </div>
+          </li>
+          <li>
+            <div class="event-li">
+              <img src="Capstone-img/blackboard.png">
+              <p class="event">Barrel racing</p>
+              <p class="event-comment">A fast-paced rodeo event where riders navigate a cloverleaf pattern around barrels, aiming for the fastest time without knocking them down.</p>
+            </div>
+          </li>
+          <li>
+            <div class="event-li">
+              <img src="Capstone-img/mesagges.png">
+              <p class="event">Tie down roping</p>
+              <p class="event-comment">A rodeo event where a cowboy on horseback ropes a calf and quickly dismounts to tie three of its legs together. The fastest time wins, emphasizing skill, speed, and precision.</p>
+            </div>
+          </li>
+          <li>
+            <div class="event-li">
+              <img src="Capstone-img/paint.png">
+              <p class="event">Team roping</p>
+              <p class="event-comment">Two riders, a header and a heeler, work together to rope a steer in a timed event.</p>
+            </div>
+          </li>
+          <li>
+            <div class="event-li">
+              <img src="Capstone-img/park.png">
+              <p class="event">Bull Riding</p>
+              <p class="event-comment">Riders attempt to stay on a bucking bull for a set time, showcasing strength and skill.</p>
+            </div>
+          </li>
+        </ul>
+        <button class="tickets">Tickets here!</button>
+        <a href="" class="join">Join our comunity</a>
+      </div>
+    </section>
+
+   <!-----------------Top standing-------------------->
+
+    <section class="top-standing">
+      <h2>Top Stanting</h2>
+      <div class="red-line"></div>
+      <div id="cowboys-container">
+        
+      </div>
+    </section>
```

Comment:
- [ ] `required`
- [ ] please add background to the images like the original template to make it look more similar

`your template`
![image](https://github.com/bohaz/capstone-project1/assets/66230306/f8024e6c-1095-46ae-8fd6-730860081940)

`original template`
![image](https://github.com/bohaz/capstone-project1/assets/66230306/d9df832d-bf80-4158-932f-2dabfea83ce8)